shard.yml

```diff
@@ -17,8 +17,8 @@ targets:
 
 dependencies:
   redis:
-    github: stefanwille/crystal-redis
-    version: ~> 2.8.0
+    github: jgaskins/redis
+    branch: master
```

Comment:
I've been working with @jgaskins on this, and there's still a few small things being updated before the next release. This will need to wait until the next release before being locked down.